DigitalLearningSolutions.Web/appsettings.json

```diff
@@ -13,5 +13,11 @@
     "RefactoredSuperAdminInterface": false,
     "UseSignposting": true
   },
-  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-dev.azurewebsites.net"
+  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-dev.azurewebsites.net",
+  "LearningHubSSO": {
+    "ToleranceInSeconds": 60,
+    "HashIterations": 10000,
+    "ByteLength": 32,
+    "SecretKey": "test value" 

```

Comment:
MapsAPIKey's value is blank in appsettings; I think we should do the same thing here to stay consistent.

Reply:
Yes good point. We can move all of these into `secrets` for local dev work. Then the actual values will need plumbing in in the pipeline. Probably good to add/update keeper with the values required to make it work in test/UAT.
Maybe the zero-to-hero wiki should be updated to reflect the need to update local `secrets` from these keeper values too?

Reply:
Thanks, updated the z2h. Not sure how secrets are set on the environments, I'll ask Kevin about it.